src/services/FirebaseAuthService.ts

```diff
@@ -0,0 +1,31 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+
+const firebaseConfig = {
+  apiKey: process.env.FIREBASE_API_KEY,
+  authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+  databaseURL: process.env.FIREBASE_DATABASE_URL,
+  projectId: process.env.FIREBASE_PROJECT_ID,
+  storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID
+};
+
+const authenticateWithFirebase = async () => {
```

Comment:
```suggestion
const authenticateWithFirebase = async (): Promise<void> => {
```

리턴 타입을 명시하는 것으로 결정하였습니다